File: codefestival_2016_qualC_a/Main.cpp
#include<iostream>
using namespace std;
int main()
{
    string str;
    cin >> str;

    bool c = false;
    string result = "No";
    for (int i = 0; i < (int)str.size(); i++) {
        if (str[i] == 'C') {
            c = true;
        } else if (str[i] == 'F' && c) {
            result = "Yes";
            break;
        }
    }
    cout << result << endl;
    return 0;
}

File: codefestival_2016_qualC_a/Main.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		String str = sc.next();
		boolean c = false;
		String result = "No";
		for (int i = 0; i < str.length(); i++) {
			if (str.charAt(i) == 'C') {
				c = true;
			} else if (str.charAt(i) == 'F' && c) {
				result = "Yes";
				break;
			}
		}
		System.out.println(result);
	}
}